SpatialGDK/Documentation/content/get-started/example-project/exampleproject-local-deployment.md

```diff
@@ -87,8 +87,9 @@ In the **Multiplayer Options** section of the window:</br>
 The **Spatial Networking** option is the networking switch; you use this to switch your game's deployment from native Unreal networking to SpatialOS networking.</br></br>
 
 
-1. Now, run the game: in the **Modes** section of the window, select **New Editor Window (PIE)**.</br> 
-You are now running one game server and two game clients in a local deployment in your Unreal Editor. (If it doesn't run, see **Note** below.)</br>
+1. Now, run the game: in the **Modes** section of the window, select **New Editor Window (PIE)**.</br>
+
+You are now running one game server and two game clients in a local deployment in your Unreal Editor. You do not need to worry about the loading bar being stuck on 45%, this is just the shaders being compiled for the first time. (If it doesn't run, see **Note** below.)</br>
```

Comment:
Worth saying how long (approx) they will be stuck?

Reply:
Agree, this would be good if possible!

Reply:
Unfortunately, I don't think there is anyway of estimating the time, it will depend on the machine.